src/components/apiproduct/APIProductForm.tsx

```
@@ -0,0 +1,487 @@
+      </div>
+
+      <ActionGroup style={{ marginTop: 'var(--pf-v6-global--spacer--lg)' }}>
+        <KuadrantCreateUpdate
+          model={getModelFromResource(constructResource())}
+          resource={constructResource()}
+          policyType="apiproduct"
+          navigate={navigate}
+          validation={isFormValid()}
+          update={isEditMode}
+        />
+        <Button variant="link" onClick={() => navigate(`/kuadrant/ns/${namespace}`)}>
```

ESLint

[error] Insert `····`

(prettier/prettier)